build.gradle

```diff
@@ -5,7 +5,7 @@ import com.synopsys.integration.detect.verification.VerifyBlackDuckDetectTask
 buildscript {
     ext {
         springBootVersion = '2.4.5'
-        blackDuckCommonVersion = '56.0.0'
+        blackDuckCommonVersion = '56.1.0'
```

Comment:
To be able to send a custom TRACKING_ID to phone-home-client.

Reply:
Which is only needed for testing. But there was something in there for Dana as well.